src/languages/scss.ts

```diff
@@ -0,0 +1,140 @@
+import {
+  cascadingMatcher,
+  conditionMatcher,
+  createPatternMatchers,
+  matcher,
+  patternMatcher,
+  trailingMatcher,
+} from "../util/nodeMatchers";
+import {
+  NodeMatcherAlternative,
+  ScopeType,
+  SelectionWithEditor,
+} from "../typings/Types";
+import { patternFinder } from "../util/nodeFinders";
+import { getNodeRange, childRangeSelector, delimitedSelector } from "../util/nodeSelectors";
+import { SyntaxNode } from "web-tree-sitter";
+
+// curl https://raw.githubusercontent.com/serenadeai/tree-sitter-scss/c478c6868648eff49eb04a4df90d703dc45b312a/src/node-types.json \
+//  | jq '[.[] | select(.type =="stylesheet") | .children.types[] | select(.type !="declaration") | .type ]'
+
+const STATEMENT_TYPES = [
+  "apply_statement",
+  "at_rule",
+  "charset_statement",
+  "debug_statement",
+  "each_statement",
+  "error_statement",
+  "for_statement",
+  "forward_statement",
+  "function_statement",
+  "if_statement",
+  "import_statement",
+  "include_statement",
+  "keyframes_statement",
+  "media_statement",
+  "mixin_statement",
+  "namespace_statement",
+  "placeholder",
+  "rule_set",
+  "supports_statement",
+  "use_statement",
+  "warn_statement",
+  "while_statement",
+];
+
+function isArgumentListDelimiter(node: SyntaxNode) {
+  return [",", "(", ")"].includes(node.type) || checkAtPlainValueDelimiter(node);
+};
+
+function checkAtPlainValueDelimiter(node: SyntaxNode) {
```

Comment:
I don't think there's any way to name this one clearly, so it could def use a quick docstring. 

I think for consistency / simplicity, though, I'd name it

```suggestion
function isAtDelimiter(node: SyntaxNode) {
```